pkg/cloud/aws/actuators/cluster/actuator.go

```diff
@@ -62,62 +62,23 @@ func (a *Actuator) Reconcile(cluster *clusterv1.Cluster) error {
 
 	ec2svc := ec2.NewService(scope)
 	elbsvc := elb.NewService(scope)
+	certSvc := certificates.NewService(scope)
 
 	// Store some config parameters in the status.
```

Comment:
does this comment still make sense?

Reply:
Updated this to describe what the reconcileCertificates() does.
Other Reconcile* are kinda self-explanatory.